sdks/python/apache_beam/io/filesystem.py

```diff
@@ -250,8 +255,73 @@ def flush(self):
 
   @property
   def seekable(self):
-    # TODO: Add support for seeking to a file position.
-    return False
+    return self._file.mode == 'r'
+
+  def _rewind_read_buffer(self):
```

Comment:
Done. I found a better name for _rewind_read_buffer: _clear_read_buffer. I think it better represents the purpose.